CONTRIBUTING.md

```diff
@@ -96,6 +96,31 @@ Some components may require specific setup steps. Please refer to their respecti
 - [Node.js Binding](bindings/nodejs/CONTRIBUTING.md)
 - [Python Binding](bindings/python/CONTRIBUTING.md)
 
+## Build from source
+
+OpenDAL uses [Cargo](https://doc.rust-lang.org/cargo/) as its build system. 
+To build OpenDAL, run the following command under OpenDAL's root directory:
+
+Only build the core component:
+
+```shell
+cargo build
+```
+
+With full features enabled:
+
+```shell
+cargo build --all-features
+```
+
+Build all components:
```

Comment:
Those components can't be used at the same time. I don't think it's worth to build all components unless in CI.

Reply:
I added this part for verifying the release candidate. But it seems that it's not the right place here. I'll add a separate verify doc.

Reply:
Removed.